udf-examples/pom.xml

```diff
@@ -117,6 +118,7 @@
                       <arg value="-DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}"/>
                       <arg value="-DGPU_ARCHS=${GPU_ARCHS}"/>
                       <arg value="-DPER_THREAD_DEFAULT_STREAM=${PER_THREAD_DEFAULT_STREAM}"/>
+                      <arg value="-DCUDF_ENABLE_ARROW_S3=${CUDF_ENABLE_ARROW_S3}"/>
```

Comment:
Refer to : cmake build flag CUDF_ENABLE_ARROW_S3 to conditionally build Arrow with S3 filesystem support.
https://github.com/rapidsai/cudf/pull/8531